frontend/src/components/ApiKeysTable/index.tsx

```diff
@@ -34,9 +34,18 @@ export const ApiKeysTable = ({
 
   const nonDeletedApiKeys = apiKeys?.filter((apiKey) => !apiKey.deletedAt);
 
+  const onSuccess = useCallback(() => {
+    closeCreationModal();
+    window.location.reload();
```

Comment:
Why do we need to force refresh the page?
There is likely a better way to do this with NextJS (`router.reload()`)

Reply:
The method router.refresh() [doesn't reload the site it just refetch the page](https://github.com/vercel/next.js/discussions/62146#discussioncomment-8493677) and merges the new response with the previous one.

For that mechanism to actually reload the page, the API request should be on the server rather than in the client. That's not the case for some components of the application. I'll add an issue to update this structure but I think it'd be better to update it in another PR.